phonebook-ui/src/app/common/service/user.service.ts

```diff
@@ -1,24 +1,34 @@
-import {Injectable} from '@angular/core';
-import {HttpClient} from "@angular/common/http";
-import {User} from "../model/user";
+import { Injectable } from '@angular/core'
+import { HttpClient } from '@angular/common/http'
+import { Observable } from 'rxjs'
+import { User } from '../interface/user.interface'
 
 @Injectable({
   providedIn: 'root'
 })
 export class UserService {
 
-  private host = 'http://localhost:8080/';
-  private registerUrl = `${this.host}api/v1/user/registration`;
-  private registerConfirmUrl = `${this.host}api/v1/user/confirmation?token=`;
+  private host = 'http://localhost:8080/'
```

Comment:
if we use proxi.conf.json , we dont need this var

Reply:
It's not my code, I didn't touch it!

Reply:
And what?=))